README.md

```diff
@@ -1,36 +1,70 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15555388&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
 
+software engineering is the systematic application of engineering principles, methods and tools to the development and maintenance of high-quality software principles.
+software engineering helps in the creating of software applications and systems that power various aspects of modern life such as transport and logistics, healthcare, banking and finance and etc.
+
 
 Identify and describe at least three key milestones in the evolution of software engineering.
 
+three key milestones in the evolution of software engineering are: 
+(i) the development of programming languages eg Fortan,C
+(ii) the establishemetn of software engineering as a discpline in the 1960s
+(iii) the rise of agile methodologies in the 2000s
 
 List and briefly explain the phases of the Software Development Life Cycle.
 
+(1)Requirements - gathering user needs and documenting system requirements 
+(2)Design -creating high level and detailed designs of the software architecture and user interface
+(3)Implementation - code writng and building the software according to the software specifications
+(4)Testing - conducting tests and ensuring software meets quality standards and functional requirements 
+(5)Deployment - relasing the software to the customers or users
+(6)Maintenance - provising ongoing support, updates, and enhancments to the software after deployment
 
 Compare and contrast the Waterfall and Agile methodologies. Provide examples of scenarios where each would be appropriate.
 
+Waterfall methodologies is a sequential approach with distinct phases eg requirements, design, implementation. flowing downwards like a waterfall. whereas agile is and iterative and incremental approach focues on flexiblity, collaboration and responding to change 
 
```

Comment:
This is an art of providing concise, precise and accurate commands to get the best response from AI tools